app/api/v2/handlers/fact_api.py

```diff
@@ -129,9 +132,11 @@ async def add_relationships(self, request: web.Request):
             self.log.warning(error_msg)
             raise JsonHttpBadRequest(error_msg)
 
-    @aiohttp_apispec.docs(tags=['facts'])
-    @aiohttp_apispec.response_schema(FactSchema)
+    @aiohttp_apispec.docs(tags=['facts'],
+                          summary='Delete a fact',
+                          description='Delete a fact by criteria provided in the `FactSchema`.')
     @aiohttp_apispec.request_schema(FactSchema(partial=True))
+    @aiohttp_apispec.response_schema(FactSchema, description='Returns the fact that was deleted in the `FactSchema` format.')
```

Comment:
This can potentially delete multiple facts, so this can be updated to denote that any facts matching the criteria may be returned